_rules/autocomplete-valid-73f2c2.md

```diff
@@ -19,7 +19,7 @@ authors:
 
 ## Applicability
 
-The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is not empty (""), except if one of the following is true:
+The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is neither empty ("") nor only [whitespace](#whitespace), except if one of the following is true:
```

Comment:
I think @dd8 has a very valid point and that will require changes…
https://github.com/act-rules/act-rules.github.io/pull/791#issuecomment-524865504

In short, `autocomplete="&nbsp &nbsp"` is "only whitespace" per our definition of whitespace, but composed of two (invalid) autocomplete values (each being non-breaking space) as per autocomplete specs (break tokens on ASCII spaces (TAB/LF/FF/CR/SPACE)).  Thus, this should be a Failure (autocomplete is not valid) rather than a Inapplicable.